src/classifier.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable security/detect-object-injection */
+// Disabling this since the object from which we are accessing the properties won't have any prototype chain.
+// Also, since we are just using this object to access properties, its safe to disable security check for now.
+
+import { generateClassifierPath } from './Helpers/ClassifierHelpers';
+
+const breaking = 'breaking';
+const nonBreaking = 'non-breaking';
+const unclassified = 'unclassified';
+
+type ChangeTypes = typeof breaking | typeof nonBreaking | typeof unclassified;
```

Comment:
You can make for it enum type :)

Reply:
I will refactor these types when we will move all the types in a `types.ts` file. So, I will convert this to an enum that time. Is it alright?

Reply:
Ok, no problem :)